src/test/spec/auth.rs

```diff
@@ -50,13 +50,7 @@ async fn run_auth_test(test_file: TestFile) {
     for mut test_case in test_file.tests {
         test_case.description = test_case.description.replace('$', "%");
 
-        let skipped_mechanisms = [
-            "GSSAPI",
-            "MONGODB-X509",
-            "PLAIN",
-            "MONGODB-CR",
-            "MONGODB-AWS",
-        ];
+        let skipped_mechanisms = ["GSSAPI", "MONGODB-X509", "PLAIN", "MONGODB-CR"];
```

Comment:
For some reason there wasn't any TODO for AWS auth even though it was listed in this array.